std-bits/table/src/main/resources/META-INF/native-image/org/enso/table/reflect-config.json

```diff
@@ -0,0 +1,1912 @@
+[
+    {
+        "name": "java.beans.PropertyChangeSupport",
+        "methods": [
+            {
+                "name": "<init>",
+                "parameterTypes": [
+                    "java.lang.Object"
+                ]
+            }
+        ]
+    },
+    {
+        "name": "org.enso.table.aggregations.Aggregator"
+    },
+    {
+        "name": "org.enso.table.aggregations.Concatenate"
+    },
+    {
+        "name": "org.enso.table.aggregations.Count"
+    },
+    {
+        "name": "org.enso.table.aggregations.CountDistinct"
+    },
+    {
+        "name": "org.enso.table.aggregations.CountEmpty"
+    },
+    {
+        "name": "org.enso.table.aggregations.CountNothing"
+    },
+    {
+        "name": "org.enso.table.aggregations.First"
+    },
+    {
+        "name": "org.enso.table.aggregations.GroupBy"
+    },
+    {
+        "name": "org.enso.table.aggregations.Last"
+    },
+    {
+        "name": "org.enso.table.aggregations.Mean"
+    },
+    {
+        "name": "org.enso.table.aggregations.MinOrMax"
+    },
+    {
+        "name": "org.enso.table.aggregations.Mode"
+    },
+    {
+        "name": "org.enso.table.aggregations.Percentile"
+    },
+    {
+        "name": "org.enso.table.aggregations.ShortestOrLongest"
+    },
+    {
+        "name": "org.enso.table.aggregations.StandardDeviation"
+    },
+    {
+        "name": "org.enso.table.aggregations.Sum"
+    },
+    {
+        "name": "org.enso.table.data.column.operation.CountNothing"
+    },
+    {
+        "name": "org.enso.table.data.column.operation.UnaryOperation"
+    },
+    {
+        "name": "org.enso.table.data.column.operation.cast.CastProblemAggregator"
+    },
+    {
+        "name": "org.enso.table.data.column.operation.cast.ConversionFailure"
```

Comment:
I would expect these classes to be listed somewhere in `polyglot java import`... Are then not? Then we solved this situation with [Extra_Imports](https://github.com/enso-org/enso/blob/7029c4ad70d1e85344412d69ce85d4ef2abb35f3/distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Extra_Imports.enso#L4) to make them more approachable to libs guys.  Listing them here does the same thing, but I don't think libs guys will ever edit this file.

Reply:
Many of these have been generated by the agent.